### PR TITLE
Pool size & Replica config

### PR DESCRIPTION
**Pull request by smathangi**

### CODE REVIEW

File: charts/ccd-data-store-api/values.aat.template.yaml [hunk 1]
```diff
@@ -1,5 +1,4 @@
 java:
-  replicas: 2
```
Comment (mario-paniccia) [inline charts/ccd-data-store-api/values.aat.template.yaml]: shouldn't AAT be an exact replica of Prod?

File: charts/ccd-data-store-api/values.yaml [hunk 2]
```diff
@@ -20,6 +20,7 @@ java:
     DATA_STORE_DB_HOST: ccd-data-store-api-postgres-db-{{ .Values.global.environment }}.postgres.database.azure.com
     DATA_STORE_DB_USERNAME: ccd@ccd-data-store-api-postgres-db-{{ .Values.global.environment }}
     DATA_STORE_DB_OPTIONS: "?stringtype=unspecified&sslmode=require"
+    DATA_STORE_DB_MAX_POOL_SIZE: 48
```
Comment (mario-paniccia) [inline charts/ccd-data-store-api/values.yaml]: Can we double check in AAT this has the same value as Prod?

File: charts/ccd-data-store-api/values.aat.template.yaml (hunk 1 above)
Comment (smathangi) [reply charts/ccd-data-store-api/values.aat.template.yaml]: This config is staging pod only which is used by functional tests only.

File: charts/ccd-data-store-api/values.yaml (hunk 2 above)
Comment (smathangi) [reply charts/ccd-data-store-api/values.yaml]: yes, it's 48 for aat as well in terraform.

File: charts/ccd-data-store-api/values.aat.template.yaml (hunk 1 above)
Comment (adusumillipraveen) [reply charts/ccd-data-store-api/values.aat.template.yaml]: yes, aat.template.yaml is used only for staging deployment where tests are run. Its a bit redundant config which we will try to get rid of soon.

Comment (mario-paniccia) [reply charts/ccd-data-store-api/values.aat.template.yaml]: 👍 That'd be great @adusumillipraveen if you can remove it. Much less maintainability effort

**Pull request merged**